lib/game.rb

```diff
@@ -0,0 +1,92 @@
+require_relative '../lib/board.rb'
+
+class Game
+
+  attr_accessor :player_1, :player_2
+
+  WIN_COMBINATIONS= [
+      [0,1,2],  #top row
+      [3,4,5], #middle row
+      [6,7,8],  #bottom row
+      [0,3,6],  #left column
+      [1,4,7],  #middle column
+      [2,5,8],  #right column
+      [0,4,8],  #diagonal 1
+      [6,4,2]   #diagonal 2
+  ]
+
+  def initialize(player_1 =  Human.new('X'), player_2 = Human.new('O'), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def cells=(set_cells)
+    @board.cells = set_cells
+  end
+
+  def cells
```

Comment:
I think this method should just be called `board` and still read out of `@cells` and you wouldn't have needed to change your test suite in 100 places.